### PR TITLE
Add Entrust to the list of available ACME endpoints

### PR DESCRIPTION
**Pull request by treidel**
Entrust CA is supported by acmebot.   This update adds the Entrust ACME endpoint to the list of available options.  Fixes shibayan/keyvault-acmebot#697

### CODE REVIEW

File: azuredeploy.bicep [hunk 1]
```diff
@@ -14,6 +14,7 @@ param mailAddress string
   'https://api.buypass.com/acme/directory'
   'https://acme.zerossl.com/v2/DV90/'
   'https://dv.acme-v02.api.pki.goog/directory'
+  'https://acme.entrust.net/acme2/directory'
```
Comment (shibayan) [inline azuredeploy.bicep:17]: @treidel Is this endpoint common to all users?

Comment (treidel) [reply azuredeploy.bicep:17]: Yes, the endpoint listed is common to all users.  When an EAB is issued it is tied to a specific customer's account, so there was not a need to provide unique endpoint URLs.

**Comment by treidel** [comment]
@shibayan is there a way for me to resolve the lint error in the build?  It seems to relate to warnings unrelated to my changes.

**Review by shibayan — APPROVED** [review]

**Comment by shibayan** [comment]
@treidel Thanks for your contribution, we have fixed the bicep error.

**Pull request merged**